data/tools/incident_response.json

```
@@ -137,6 +137,15 @@
       "language": "Go",
       "price": "Free",
       "online": "False"
-    }
+    },
+    {
+      "name": "Snort",
+      "website": "https://www.snort.org/",
+      "source": "https://github.com/snort3/snort3",
+      "description": "An open-source intrusion detection system (IDS) that monitors network traffic for suspicious activities and threats.",
```

```suggestion
      "description": "Intrusion detection system (IDS) that monitors network traffic for suspicious activities and threats",
```

> Description begins with an uppercase letter and doesn’t end with a dot (.)

https://inventory.raw.pm/docs/docs/contribution_guidelines/format.html